tests/vm_test.go

```diff
@@ -269,38 +270,38 @@ var _ = Describe("VirtualMachine", func() {
 			var err error
 
 			By("Creating a new VMI")
-			newVMI := newVirtualMachine(true)
+			newVM := newVirtualMachine(true)
 
 			// wait for a running VirtualMachineInstance.
 			By("Waiting for the VMI's VirtualMachineInstance to start")
 			Eventually(func() error {
-				firstVMI, err = virtClient.VirtualMachineInstance(newVMI.Namespace).Get(newVMI.Name, &v12.GetOptions{})
+				firstVMI, err = virtClient.VirtualMachineInstance(newVM.Namespace).Get(newVM.Name, &v12.GetOptions{})
 				if err != nil {
 					return err
 				}
 				if !firstVMI.IsRunning() {
 					return fmt.Errorf("vmi still isn't running")
 				}
 				return nil
-			}, 120*time.Second, 1*time.Second).Should(Succeed())
+			}, 120*time.Second, 5*time.Second).Should(Succeed(), "VMI should have started")
```

Comment:
Did you observe specific issues here? Is one call per second really too much?

Reply:
I have two reasons for this setup. First there is simply no need to check it faster. The VM takes time to boot. Second, I am thinking of parallel tests. I do not want to DOS API when running in parallel.

Do you agree?